bin/setup

```diff
@@ -28,6 +28,13 @@ if ! command -v psql > /dev/null; then
   exit -1
 fi
 
+if ! command -v elasticsearch > /dev/null; then
+  printf "${CLEAR_LINE}${RED}  You must install elasticsearch before setup can continue${NO_COLOR}\n"
+  printf "    You should:\n"
+  printf "      $ brew install elasticsearch\n"
```

Comment:
Is there a certain version of ES that should be installed. I've got 2.4 installed but I think latest is 5, correct? I assume brew would give you the latest version by default but it's been a while since I was messing around with this.